src/coreclr/System.Private.CoreLib/src/System/MulticastDelegate.cs

```diff
@@ -595,15 +595,15 @@ private void CtorRTClosed(object target, IntPtr methodPtr)
         }
 
         [System.Diagnostics.DebuggerNonUserCode]
-        private void CtorOpened(object target, IntPtr methodPtr, IntPtr shuffleThunk)
+        private void CtorOpened(object _, IntPtr methodPtr, IntPtr shuffleThunk)
```

Comment:
I do not think it is an improvement to omit the argument name in cases like this one where the method signature has to be symmetric with other method signatures.

(I am not commenting on all instances of this issue.)

Reply:
ok, moved to pragma